src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int maxRandomNumber = 100;
+        int randomNumber = new Random().nextInt(maxRandomNumber);
```

Comment:
Creating a new instance of Random every time the method is called is not efficient. You should create a single instance as a class-level field and reuse it. Refer to the checklist item about not creating several instances of the same class if you can use only one instance for your purposes.